lib/rules/web/admin_console/4.10/logs.xyaml

```diff
@@ -95,12 +95,11 @@ check_event_is_streaming:
 check_logs_not_wrapped:
   elements:
   - selector:
-      xpath: //div[contains(@class,'log-window__lines--wrap')]
-    missing: true
+      xpath: //div[contains(@style,'auto')]
```

Comment:
`sytle` property seems unreliable

Reply:
I could not find other element/attr to reflect the wrap change better.

Reply:
me too :  (